README.md

```diff
@@ -1,39 +1,31 @@
 # Cardano Development Updates
 
-This repository will host updates from the
-[cardano](https://github.com/input-output-hk/cardano-node) development teams.
-The generated website containing the updates can be found
-[here](https://input-output-hk.github.io/cardano-updates/).
+This repository will host updates from the [cardano](https://github.com/input-output-hk/cardano-node) development teams.
```

Comment:
Minor: I'd rather keep the line length to ~80 chars. 

If we wanted to change this anyway maybe we could have made it a separate PR to minimise the diff.

Reply:
Good point, I usually don't care but I am fine sticking to that 80 chars limit.